.github/workflows/ci.yml

```diff
@@ -40,24 +42,43 @@ jobs:
                 CHANGED=""
```

Comment:
If test with `@Tag("long-running")` has been modified together with a root project POM file, this long running test will not be tested, right?

Reply:
Yes, for this version. I am working on adding support to run all modules include long-running in [my forked project](https://github.com/gtroitsk/quarkus-test-suite/pull/4/files) - you can expect something like that

Reply:
ok, just request review again when you are done.

for record, I think this is PR is fine, I think we don't need perfect version because IMHO it is super edge case. personally, I'd much more prefer less verbose version that drops `excludedGroups` when it detects changes in at least one module with a long running tag and that would be it. IMO bash in the workflow is hard to maintain, debug and keep consistent. anyway, I'll go with whatever you bring as long as it works.